README.md

```diff
@@ -1,2 +1,107 @@
-# eslint-udemy
-ESLint configs and plugins used by Udemy.
+Udemy ESLint Packages
+=====================
+
+This repository has [*ESLint*](https://www.github.com/eslint/eslint) related packages used by various Udemy projects. 
+It is owned by [@udemy/team-f](https://github.com/orgs/udemy/teams/team-f), but everyone is welcome to contribute.
+Please refer to [Contributing](#contributing) section to learn more.
+
+This repository uses [*Lerna*](https://github.com/lerna/lerna) to manage multiple [*npm*](https://www.npmjs.com/) packages. We 
+configured Lerna to use [*Yarn Workspaces*](https://yarnpkg.com/lang/en/docs/workspaces/) under the hood.
+
+# Installation
+
+Refer to individual packages' *Installation* section in order to learn more about how you can use each package.
```

Comment:
Refer to each package's Installation section in order to learn how to install each package.